eng/Version.Details.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.DotNet.Cli.Runtime" Version="3.0.100-preview4.19126.2">
+    <Dependency Name="Microsoft.DotNet.Cli.Runtime" Version="3.0.100-preview.19127.2">
```

Comment:
This got reverted to preview in a merge conflict resolution error. Sent https://github.com/dotnet/cli/pull/10893 to fix it. 

We don't need to block this PR on that as this version doesn't show in the product.